cpp/src/ray/api.cc

```diff
@@ -34,6 +37,12 @@ void Init() {
   Init(config, nullptr, nullptr);
 }
 
-void Shutdown() { ray::internal::AbstractRayRuntime::DoShutdown(); }
+bool IsInitialized() { return is_inited_; }
```

Comment:
Why not call something like `ray::internal::RayRuntimeHolder::Instance().IsInitialized()` and remove the `is_inited_` field?

Reply:
Good for me.